project-management/src/main/java/life/qbic/projectmanagement/application/measurement/MeasurementService.java

```diff
@@ -342,7 +236,8 @@ public Result<MeasurementId, ErrorCode> update(ProjectId projectId,
   }
 
   private Result<MeasurementId, ErrorCode> updatePxP(ProteomicsMeasurementMetadata metadata) {
-    List<DomainEvent> domainEventsCache = new ArrayList<>();
+
+    /*List<DomainEvent> domainEventsCache = new ArrayList<>();
```

Comment:
Is this code commented out intentionally? Can it be removed? Where did the functionality of updating PxP go?